### ISSUE TITLE
selecting a parent view(rootView)

### ISSUE THREAD

**Issue by mignick** [nicolaschengdev/WYPopoverController#46]
Thanks for a great control! 
It seems to me there is a small issue in

``` objectivec
- (UIView *)rootView
{
    UIWindow *result = [[UIApplication sharedApplication] keyWindow];

    if (result.subviews.count > 0)
    {
        result = [result.subviews lastObject];
    }

    return result;
}
```

It's not good idea to select blindly last view from a window as an parent. At least it should be checked for visibility etc. But I think to move rootView property to the public interface is a better approach: if a user does not set this property then select something from window's subviews.

Thanks!

**Comment by nicolaschengdev** [comment]
It seems you are in a special case and the best practice would be to remove any hidden views I think.

I will experiment an another implementation because your solution doesn't follow UIPopoverController protocol and it minds me.

**Comment by nicolaschengdev** [comment]
fixed with issue #47
